website/photos/models.py

```diff
@@ -44,7 +44,9 @@ class Photo(models.Model):
         "Album", on_delete=models.CASCADE, verbose_name=_("album")
     )
 
-    file = ImageField(_("file"), upload_to=photo_uploadto)
+    file = ImageField(
+        _("file"), upload_to=photo_uploadto, pregenerated_sizes=["medium"]
```

Comment:
```suggestion
        _("file"), upload_to=photo_uploadto, pregenerated_sizes=["small", "medium"]
```
The app uses small

Reply:
And actually, since thumbnails are generated before creating the URL, they all need to be pregenerated: the API returns small, medium, large, and full urls.